src/prompt/cursor.rs

```diff
@@ -130,6 +135,10 @@ impl StringCursor {
     pub fn iter_mut(&mut self) -> impl Iterator<Item = &mut char> {
         self.value.iter_mut()
     }
+
+    pub fn value_to_string(&self) -> String {
+        self.value.iter().collect()
+    }
```

Comment:
It's not needed, the cursor already implements `to_string()` because `Display` trait is implemented.

Reply:
Oh I did not know that ! Thanks :)